accounts/views.py

```diff
@@ -1,6 +1,427 @@
+
+def _user_base_payload(user):
+    """기본 사용자 정보 JSON (라벨 변환)"""
+    return {
+        "email": user.email,
+        "name": user.name,
+        "birth": user.birth,
+        "sex": _labels_from_choices(SexChoices, user.sex),
+        "is_marketing_allowed": user.is_marketing_allowed,
+    }
+
+
+# ── Views ──────────────────────────────────────────────────────────────────
+class AccountsLoginRoot(APIView):
+    """
+    POST /accounts/login
+    """
+    permission_classes = [AllowAny]
+
+    def post(self, request):
+        serializer = UserLoginSerializer(data=request.data, context={"request": request})
+        serializer.is_valid(raise_exception=True)
+        user = serializer.validated_data["user"]
+
+        # JWT 발급
+        refresh = RefreshToken.for_user(user)
+        access = refresh.access_token
+
+        # 만료 시각 (exp → ISO8601 'Z')
+        access_exp = datetime.fromtimestamp(access["exp"], tz=dt_timezone.utc)
+        refresh_exp = datetime.fromtimestamp(refresh["exp"], tz=dt_timezone.utc)
+
+        profiles = []
+        if Proposer.objects.filter(user=user).exists():
+            profiles.append("proposer")
+        if Founder.objects.filter(user=user).exists():
+            profiles.append("founder")
+
+        body = {
+            "grant_type": "Bearer",
+            "access": {
+                "token": str(access),
+                "expire_at": access_exp.isoformat().replace("+00:00", "Z"),
+            },
+            "refresh": {
+                "token": str(refresh),
+                "expire_at": refresh_exp.isoformat().replace("+00:00", "Z"),
+            },
+            "profile": profiles,
+        }
+        return Response(body, status=status.HTTP_200_OK)
+
+
+class AccountsAccessTokenRoot(APIView):
+    """
+    POST /accounts/access-token
+    """
+    permission_classes = [AllowAny]
+
+    def post(self, request):
+        req = JwtRefreshRequestSerializer(data=request.data)
+        req.is_valid(raise_exception=True)
+
+        refresh_str = req.validated_data["refresh_token"]
+
+        sjw = SJWTokenRefreshSerializer(data={"refresh": refresh_str})
+        sjw.is_valid(raise_exception=True)
+        sjw_data = sjw.validated_data  # {'access': '...', ['refresh': '...']}
+
+        access_str = sjw_data["access"]
+        new_refresh_str = sjw_data.get("refresh", refresh_str)
+
+        access_exp = datetime.fromtimestamp(
+            AccessToken(access_str)["exp"], tz=dt_timezone.utc
+        ).isoformat().replace("+00:00", "Z")
+
+        refresh_exp = datetime.fromtimestamp(
+            RefreshToken(new_refresh_str)["exp"], tz=dt_timezone.utc
+        ).isoformat().replace("+00:00", "Z")
+
+        body = {
+            "grant_type": "Bearer",
+            "access": {
+                "token": access_str,
+                "expired_at": access_exp,
+            },
+            "refresh": {
+                "token": new_refresh_str,
+                "expired_at": refresh_exp,
+            },
+        }
```

Comment:
❗ API 명세서가 잘못되어 있었어요!! `expire_at`으로 맞추는 게 맞습니다 수정 부탁드려요!
```suggestion
        body = {
            "grant_type": "Bearer",
            "access": {
                "token": access_str,
                "expire_at": access_exp,
            },
            "refresh": {
                "token": new_refresh_str,
                "expire_at": refresh_exp,
            },
        }
```